integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -41,4 +43,50 @@ def get_context_data(self, **kwargs):
                 ),
             }
         )
+        context.update(self.get_hix_context())
         return context
+
+    def get_hix_context(self):
+        """
+        Extend context by HIX info
+
+        :return: The HIX context dictionary
+        :rtype: dict
+        """
+        if not settings.TEXTLAB_API_ENABLED:
+            return {}
+
+        # Get the current region
+        region = self.request.region
+        if not region.hix_enabled:
+            return {}
+
+        # Get all pages of this region which are considered for the HIX value
+        hix_pages = region.get_pages(return_unrestricted_queryset=True).filter(
+            hix_ignore=False
+        )
+        # Get the latest versions of the page translations for these pages
+        hix_translations = PageTranslation.objects.filter(
+            language__slug__in=settings.TEXTLAB_API_LANGUAGES, page__in=hix_pages
+        ).distinct("page_id", "language_id")
+
+        # Get all hix translations where the score is set
+        hix_translations_with_score = [pt for pt in hix_translations if pt.hix_score]
+
+        # Get the worst n pages
+        worst_his_translations = sorted(
+            hix_translations_with_score, key=lambda pt: pt.hix_score, reverse=True
```

Comment:
```suggestion
            hix_translations_with_score, key=lambda pt: pt.hix_score
```
Sorry, my suggestion was incorrect here: `sorted()` already sorts ascending by default, the `reverse` caused a descending order...